testing/file_test/file_test_base.h

```diff
@@ -166,10 +162,9 @@ struct FileTestFactory {
 extern auto GetFileTestFactory() -> FileTestFactory;
 
 // Provides a standard GetFileTestFactory implementation.
-#define CARBON_FILE_TEST_FACTORY(Name)                                         \
-  auto GetFileTestFactory()->FileTestFactory {                                 \
-    return {(#Name),                                                           \
-            [](const std::filesystem::path& path) { return new Name(path); }}; \
+#define CARBON_FILE_TEST_FACTORY(Name)                                     \
+  auto GetFileTestFactory()->FileTestFactory {                             \
+    return {(#Name), [](llvm::StringRef path) { return new Name(path); }}; \
```

Comment:
```suggestion
#define CARBON_FILE_TEST_FACTORY(Name)                                   \
  auto GetFileTestFactory()->FileTestFactory {                           \
    return {#Name, [](llvm::StringRef path) { return new Name(path); }}; \
```
(Trivial nit, no need for parens around a string literal.)

Reply:
Done